lib/cfa/base_model.rb

```diff
@@ -53,14 +53,15 @@ def load
     # smart to at first modify existing value, then replace commented out code
     # and if even that doesn't work, then append it at the end
     # @note prefer to use specialized methods of children
-    def generic_set(key, value)
-      modify(key, value) || uncomment(key, value) || add_new(key, value)
+    def generic_set(key, value, tree = data)
```

Comment:
We are changing the API substantially but neglecting to change the documentation comments.

It does not make sense to me: This is an interface for some more or less abstract concept, but we torpedo it by allowing to inject the internal representation to act on? So that the actual state of the receiver object does not have to change? I don't understand at all. Please explain and link to context. (Grumble, I even had to fill in the empty PR description.)

AFAICT the rest of the PR does not even use this commit.